testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -5741,6 +5741,19 @@ private static Matcher<SqlNode> isCharLiteral(String s) {
             + "FROM (VALUES (ROW(TRUE))))");
   }
 
+  @Test void testOracleConvert() {
```

Comment:
Would it be better to change it to testConvertOracle? Other similar method names and comments need to be consistent.

Reply:
Done.